docs-next/pages/tutorials/getting-started-with-create-keystone-next-app/assets/cover.svg

```diff
@@ -0,0 +1,67 @@
+			<tspan x="226" y="729" fill="#80B1FF">λ</tspan>
+			<tspan x="346.410156" y="729" font-family="Monaco" fill="#FFFFFF"> yarn create keystone-next-app</tspan>
+			<tspan x="226" y="979" font-family="Monaco" fill="#FFFFFF">yarn create v1.22.10</tspan>
+			<tspan x="226" y="1229" font-family="Monaco" fill="#A9A9A9">[1/4]</tspan>
+			<tspan x="826.097656" y="1229" font-family="Monaco" fill="#FFFFFF"> </tspan>
+			<tspan x="946.117188" y="1229" font-family="AppleColorEmoji, Apple Color Emoji" fill="#FFFFFF">🔍</tspan>
+			<tspan x="1146.11719" y="1229" font-family="Monaco" fill="#FFFFFF">  Resolving packages...</tspan>
+			<tspan x="226" y="1492" font-family="Monaco" fill="#A9A9A9">[2/4]</tspan>
+			<tspan x="826.097656" y="1492" font-family="Monaco" fill="#FFFFFF"> </tspan>
+			<tspan x="946.117188" y="1492" font-family="AppleColorEmoji, Apple Color Emoji" fill="#FFFFFF">🚚</tspan>
+			<tspan x="1146.11719" y="1492" font-family="Monaco" fill="#FFFFFF">  Fetching packages...</tspan>
+			<tspan x="226" y="1755" font-family="Monaco" fill="#A9A9A9">[3/4]</tspan>
+			<tspan x="826.097656" y="1755" font-family="Monaco" fill="#FFFFFF"> </tspan>
+			<tspan x="946.117188" y="1755" font-family="AppleColorEmoji, Apple Color Emoji" fill="#FFFFFF">🔗</tspan>
+			<tspan x="1146.11719" y="1755" font-family="Monaco" fill="#FFFFFF">  Linking dependencies...</tspan>
+			<tspan x="226" y="2018" font-family="Monaco" fill="#A9A9A9">[4/4]</tspan>
+			<tspan x="826.097656" y="2018" font-family="Monaco" fill="#FFFFFF"> </tspan>
+			<tspan x="946.117188" y="2018" font-family="AppleColorEmoji, Apple Color Emoji" fill="#FFFFFF">🔨</tspan>
+			<tspan x="1146.11719" y="2018" font-family="Monaco" fill="#FFFFFF">  Building fresh packages...</tspan>
+			<tspan x="226" y="2281" font-family="Monaco" fill="#46FF00">success</tspan>
+			<tspan x="1066.13672" y="2281" font-family="Monaco" fill="#FFFFFF"> Installed "create-keystone-next-app@1.1.0" with binaries:</tspan>
+			<tspan x="226" y="2531" font-family="Monaco" fill="#FFFFFF">&#160;&#160;&#160;&#160;&#160;&#160;- create-keystone-next-app</tspan>
+			<tspan x="226" y="2781" font-family="Monaco" fill="#FFFFFF">[#######################################################################] 140/140</tspan>
+			<tspan x="226" y="3031" fill="#46FF00">✔</tspan>
+			<tspan x="346.410156" y="3031" font-family="Monaco" fill="#FFFFFF" font-weight="bold"> What directory should create-keystone-next-app generate your app into? · </tspan>
+			<tspan x="9227.85547" y="3031" font-family="Monaco" fill="#46FF00">my-app</tspan>
+			<tspan x="9947.97266" y="3031" font-family="Monaco" fill="#FFFFFF"></tspan>
+			<tspan x="226" y="3281" fill="#46FF00">✔</tspan>
+			<tspan x="346.410156" y="3281" font-family="Monaco" fill="#FFFFFF" font-weight="bold"> What database url should we use? · </tspan>
+			<tspan x="4667.11328" y="3281" font-family="Monaco" fill="#46FF00">postgres://localhost/db</tspan>
```

Comment:
I'm pretty sure Prisma won't work without a username and password in the connection string

```suggestion
			<tspan x="4667.11328" y="3281" font-family="Monaco" fill="#46FF00">postgres://dbuser:dbpass@localhost/db</tspan>
```

Reply:
I have tried this and it works with the default username without a password which happens to be the default install with brew

Reply:
Hmm, I wonder if this was something they fixed in 2.19.0? This definitely used to break, and was a pain the butt, so good if it's fixed 🚀